src/app/components/media/MediaComponentRightPanel.js

```diff
@@ -90,7 +90,7 @@ const MediaComponentRightPanel = ({ projectMedia, setShowTab, showTab }) => {
       { /* Set maxHeight to screen height - (media bar + tabs) */ }
       <Box maxHeight="calc(100vh - 112px)" style={{ overflowY: 'auto' }}>
         { showTab === 'requests' ? <MediaRequests media={projectMedia} all={!projectMedia.is_confirmed_similar_to_another_item} /> : null }
-        { showTab === 'suggestedMedia' ? <MediaSuggestions projectMedia={projectMedia} /> : null }
+        { showTab === 'suggestedMedia' ? <MediaSuggestions dbid={projectMedia.dbid} teamDbid={projectMedia.team?.dbid} /> : null }
```

Comment:
We no longer pass the entire `projectMedia` object into `MediaSuggestions` -- all it requires is the `dbid` and the team `dbid` and then `MediaSuggestions` and `PaginatedMediaSuggestions` plug those in as variables to all their own graphql queries.